lxd/instance/drivers/driver_qemu.go

```diff
@@ -1221,7 +1221,17 @@ func (d *qemu) start(stateful bool, op *operationlock.InstanceOperation) error {
 	// Define a set of files to open and pass their file descriptors to qemu command.
 	fdFiles := make([]*os.File, 0)
 
-	confFile, monHooks, err := d.generateQemuConfigFile(mountInfo, qemuBus, devConfs, &fdFiles)
+	// If user has requested AMD SEV, check if supported and add to QEMU config.
+	sevOpts := &qemuSevOpts{}
```

Comment:
Is it possible to move the call to d.setupSEV into d.generateQemuConfigFile, like you did with addVmgenDeviceConfig?